### PR TITLE
Exercise 13

### PR DESCRIPTION
**Pull request by barrantes-sofiaanne**
Please review my files!Tenkyu!

### CODE REVIEW

File: barrantes_sofiaanne/index.css
Comment (MentokAlmera) [inline barrantes_sofiaanne/index.css:1]: Make the indention of all css use only 2 space not 4 spaces.  

`body {
  margin: 0;
  padding: 0;
  font-family: Arial, sans serif;
}`

File: barrantes_sofiaanne/index.css [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+            "submit";
+    }
+    .split-right {
+        display: flex;
+        flex-direction: column;
+    }
+    .section-container {
+        width: auto;
+    }
+}
```
Comment (MentokAlmera) [inline barrantes_sofiaanne/index.css]: Separate every css block with a single space.

https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24acb740da7_0_35

File: barrantes_sofiaanne/index.css [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+
+.copy {
+    grid-area: copy;
+}
+
+.email, .Name, .subject, .message, .age, .copy {
+    border:3px solid white;
+    padding: 20px;
+    text-align:start;
+}
```
Comment (MentokAlmera) [inline barrantes_sofiaanne/index.css]: make it  

`.email,
.Name,
.subject,
.message,
.age,
.copy {
  border: 3px solid white;
  padding: 20px;
  text-align: start;
}`

to become more readable

File: barrantes_sofiaanne/index.css [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+
+.com {
+    grid-area: com;
+}
+
+.fname {
+    grid-area: fname;
+}
+
+.com, .fname {
```
Comment (MentokAlmera) [inline barrantes_sofiaanne/index.css]: .com,
.fname {  
  ---
  ---
  --
}

make this to become more readable

File: barrantes_sofiaanne/index.html [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+                <div class="comm">
+                    <h2>Comment Form</h2>
+                    <form id="commentForm">
+                        <div class="fname">
+                            <label for="fname">Name: </label>
+                            <input type="text" name="fname" id="fname">
+                        </div>
+                        <div class="com">
+                            <label for="comment">Comment: </label>
+                            <textarea id="comment" name="comment" rows="5" cols="33"></textarea>
```
Comment (MentokAlmera) [inline barrantes_sofiaanne/index.html]: exceed 80 cols

File: barrantes_sofiaanne/index.html [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+                        </div>
+                        <div class="com-b">
+                            <button disabled type="submit" id="com-b">Submit</button>
```
Comment (MentokAlmera) [inline barrantes_sofiaanne/index.html]: exceed 80 cols

File: barrantes_sofiaanne/index.html [hunk 6]
```diff
@@ -0,0 +1,170 @@
...
+                    <ul>
+                        <li> May your come true! -Precious </li>
+                        <li> Great Job! - Eliza</li>
+                        <li>Goodluck on your goals. :D - Mikaella</li>
+                        <li>Praying for you! - Alona</li>
+                    </ul>
+                </div>
+                <div class="comm">
+                    <h2>Comment Form</h2>
+                    <form id="commentForm">
```
Comment (MentokAlmera) [inline barrantes_sofiaanne/index.html]: IDs should be written in snake case. All lower case separated by underscore.

https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5

File: barrantes_sofiaanne/index.html [hunk 7]
```diff
@@ -0,0 +1,170 @@
...
+        </div>
+    </div>
+    <div class="contact">
+        <h2>Contact Me</h2>
+        <form>
+            <div class="email">
+                <label for="email">Email:</label>
+                <input type="email" id="email" name="email" />
+            </div>
+            <div class="Name">
```
Comment (MentokAlmera) [inline barrantes_sofiaanne/index.html]: class name should be all lower case

https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5

**Review by MentokAlmera — APPROVED** [review]
After you change your id name and class name. Modify it also in your js and css files.

**Review by MentokAlmera — CHANGES_REQUESTED** [review]
Sorry, I accidentally clicked approve, but I request some changes to your code.

**Comment by barrantes-sofiaanne** [comment]
Thank you Po!

File: barrantes_sofiaanne/index.css
Comment (MentokAlmera) [inline barrantes_sofiaanne/index.css:1]: fix all the indention of css. Make it two spaces only.

File: barrantes_sofiaanne/index.html [hunk 8]
```diff
@@ -146,23 +146,25 @@ <h2>Comment</h2>
                 </div>
                 <div class="comm">
                     <h2>Comment Form</h2>
-                    <form id="commentForm">
+                    <form id="comment-form">
```
Comment (MentokAlmera) [inline barrantes_sofiaanne/index.html]: id name is all lower case separated by underscore.

File: barrantes_sofiaanne/index.html [hunk 9]
```diff
@@ -146,23 +146,25 @@ <h2>Comment</h2>
...
-                            <button disabled type="submit" id="com-b">Submit</button>
+                            <button disabled type="submit" id="com-b">Submit
+                            </button>
                         </div>                
                     </form>
                 </div>  
                     <button id="asc_btn">Accending Order</button>
                     <button id="desc_btn">Descending Order</button>
-                <div id="commentsSection"></div>                     
+                <div id="comments-section"></div>
```
Comment (MentokAlmera) [inline barrantes_sofiaanne/index.html]: id nming is all lower case separated by underscore.

**Review by MentokAlmera — CHANGES_REQUESTED** [review]
Modify all your files again for naming. Then, for CSS, make it only two spaces for indention.

File: barrantes_sofiaanne/index.html [hunk 10]
```diff
@@ -146,7 +146,7 @@ <h2>Comment</h2>
                 </div>
                 <div class="comm">
                     <h2>Comment Form</h2>
-                    <form id="comment-form">
+                    <form id="comments-form">
```
Comment (MentokAlmera) [inline barrantes_sofiaanne/index.html]: still not following the name format for id

**Review by MentokAlmera — CHANGES_REQUESTED** [review]
please review it

**Review by MentokAlmera — APPROVED** [review]
Great, you can now merge this. <3

**Comment by barrantes-sofiaanne** [comment]
Thank you po!

**Pull request merged**